src/components/modeler/moveWithArrowKeys.js

```diff
@@ -47,3 +48,16 @@ function expandPoolToContainElement(shape) {
 
   pool.component.updateLaneChildren();
 }
+
+export function keydownListener(event) {
+  const formElementHasFocus = event.target.toString().toLowerCase().indexOf('body') === -1;
```

Comment:
I think using [`includes`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes) would be more readable than `indexOf` with a comparison to `-1`.

I wonder if we can also use `event.target === document.body`?

Reply:
Yeah that would be a lot more readable, thank you